.jenkins/Packaging.Jenkinsfile

```diff
@@ -14,7 +14,8 @@ def packageUpload(String version, String build_type) {
                            cmake ${WORKSPACE} -DCMAKE_BUILD_TYPE=${build_type} -DCMAKE_INSTALL_PREFIX:PATH='/opt/openenclave' -DCPACK_GENERATOR=DEB
                            make
                            ctest --output-on-failure --timeout ${CTEST_TIMEOUT_SECONDS}
-                           make package
+                           cpack
```

Comment:
Is this function suppose to be for Linux? Maybe it will make it clearer if this function is renamed LinuxUpload?  It would also help if there is a comment for this function describing what it is doing.

Reply:
Yes totally agree, thanks @jazzybluesea!